src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1681,6 +1716,24 @@ asf::auto_release_ptr<asr::Camera> build_camera(
                 // Create camera - DOF disabled.
                 camera = asr::PinholeCameraFactory().create("camera", params);
             }
+
+            if (phys_camera->GetMotionBlurEnabled(time, FOREVER))
+            {
+                const asf::Transformd transform =
+                    asf::Transformd::from_local_to_parent(
+                        asf::Matrix4d::make_scaling(asf::Vector3d(settings.m_scale_multiplier)) *
+                        to_matrix4d(view_node->GetObjTMAfterWSM(time + GetTicksPerFrame())));
+
+                camera->transform_sequence().set_transform(1.0, transform);
+
+                // Set motion blur parameters.
+                const float opening_time = phys_camera->GetShutterOffsetInFrames(time, FOREVER);
+                const float closing_time = opening_time + phys_camera->GetShutterDurationInFrames(time, FOREVER);
+                params.insert("shutter_open_begin_time", opening_time);
```

Comment:
Let's recap:

1. You're setting the first transform at time 0: **since you adopted "frames" as the unit of time**, that means you're setting the first transform at frame 0.

2. You're setting the second transform at time 1, i.e. at frame 1.

3. You're setting the shutter open and close times at (fractional) frame values, which I hope are between 0 and 1.

So it looks correct.